### ISSUE TITLE
iPad improvements

### ISSUE THREAD

**Issue by markrhoughton** [pocketworks/appero-sdk-ios#4]
I know this is a proof of concept for iOS, and SoonCall is certainly not optimised for iPad either, but adding this in case there's any quick styling wins for iPad

<img width="2972" alt="RocketSim_Screenshot_iPad_Pro_13-inch_(M4)_13 0_2024-10-14_15 56 22" src="https://github.com/user-attachments/assets/a2106d47-b3aa-4c9f-a006-654c9582c36c">

**Comment by roryprior** [comment]
Yep iPad wasn't on the radar when this was developed, but we should offer a better experience.
